examples/assets/scripts/utils/cubemap-renderer.js

```diff
@@ -0,0 +1,81 @@
+// How to use in the Editor:
+// - create 6 layers for rendering cubemap faces, and pass base name of these layers to .layerBaseName
+//   example: layerBaseName = "CubeLayer", layers "CubeLayer0", CubeLayer1" .. "CubeLayer5" need to be created in the Editor
+// - add your entities with Model component to these layers
+// - add this script component on an Entity, and use script assign .cubeMap to your material:
+//   material.cubeMap = entity.script.cubemapRenderer.cubeMap;
+
+var CubemapRenderer = pc.createScript('cubemapRenderer');
+
+CubemapRenderer.attributes.add('resolution', { type: 'number', default: 64 });
```

Comment:
Maybe do an enum here to restrict to power of two? 16, 32, 64, 128, 256, 512, 1024, 2048?

Reply:
well I tried and it works with any size .. I improved description to explain mipmap limitation.

Reply:
Great, sounds good.